src/cmd/create.go

```diff
@@ -678,8 +679,6 @@ func pullImage(image, release, authFile string) (bool, error) {
 		}
 	}
 
-	logrus.Debugf("Looking for image %s", imageFull)
-
```

Comment:
This debug message shouldn't be removed.

Reply:
Included again :)